bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusGenericBindingProvider.java

```diff
@@ -152,14 +153,19 @@ State getItemState() {
             return item.getState();
         }
 
+        public void setItemState(State state) {
+            ((GenericItem) item).setState(state);
+            ;
```

Comment:
Bindings should not directly set the state of items, or even hold onto references to items.